goclean.sh

```diff
@@ -5,14 +5,19 @@
 # 3. golint        (https://github.com/golang/lint)
 # 4. go vet        (http://golang.org/cmd/vet)
 # 5. test coverage (http://blog.golang.org/cover)
+#
+# gometaling (github.com/alecthomas/gometalinter) is used to run each each
+# static checker.
 
 set -e
 
 # Automatic checks
 test -z "$(gofmt -l -w .     | tee /dev/stderr)"
```

Comment:
That is in there because when I add it to gometalinter it finds an issue.  To make things clearer I left it like this in this PR and changed that one over to gometalinter in #98 along with fixing the issue it found there.